opm/parser/eclipse/EclipseState/Schedule/Group.cpp

```diff
@@ -97,7 +97,8 @@ namespace Opm {
         m_injection( new GroupInjection::InjectionData(timeMap) ),
         m_production( new GroupProduction::ProductionData( timeMap )),
         m_wells( new DynamicState< std::shared_ptr< const WellSet > >( timeMap , std::make_shared< const WellSet >() ) ),
-        m_isProductionGroup( new DynamicState<bool>(timeMap, true))
+        m_isProductionGroup( new DynamicState<bool>(timeMap, false)),
+        m_isInjectionGroup( new DynamicState<bool>(timeMap, false))
```

Comment:
When both are initialised with false we are in a situation where the well is created as a ??? well? That might be OK, but it seems a bit weird!?